skunkworks_crow/src/main/AndroidManifest.xml

```diff
@@ -83,7 +83,8 @@
 
         <activity
             android:name=".views.ui.send.SendFormsActivity"
-            android:parentActivityName=".views.ui.main.MainActivity">
+            android:parentActivityName=".views.ui.main.MainActivity"
+            android:configChanges="keyboardHidden|orientation|screenSize">
```

Comment:
@HarshitSachan1999  we should not handle screen orientation changes using manifest changes.

Reply:
@lakshyagupta21  Does this mean that I have to look for some other method to resolve it ?